### PR TITLE
ACMS-1576: Fix PHP Unit test cases

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes ACMS-1576

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
         "drupal/acquia_claro": "^1.3.4@alpha",
         "drupal/upgrade_status": "^3.18",
         "drush/drush": "^10 || ^11",
+        "nnnick/chartjs": "^3.9",
```
Comment (rajeshreeputra) [inline composer.json:24]: this should go in acquia_cms_common/composer.json

File: composer.json [hunk 2]
```diff
@@ -154,6 +155,18 @@
         "drupal": {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
+        },
+        "chart.js": {
```
Comment (rajeshreeputra) [inline composer.json:159]: this should go in acquia_cms_common/composer.json

File: modules/acquia_cms_search/tests/src/Functional/AcquiaSearchIntegrationTest.php [hunk 3]
```diff
@@ -64,35 +64,8 @@ public function testAcquiaSearchIntegration() {
...
-    // with Acquia Search Solr should be disabled, along with any views that are
-    // using it.
-    // @todo write new test as per 3.1 acquia search release.
-    /*
-    $this->assertSame('acquia_search_server',
-    Index::load('content')->getServerId());
-    $index = Index::load('acquia_search_index');
-    $this->assertFalse($index->status());
-    $this->assertNull($index->getServerId());
-    $this->assertFalse(View::load('acquia_search')->status());
```
[the remark this answers is not in the document]
Comment (deepakmishra2) [reply modules/acquia_cms_search/tests/src/Functional/AcquiaSearchIntegrationTest.php:94]: @chandan-singh7929 this is checked here AcquiaSearchIntegrationFromTourPageTest.php in acquia_cms_search test

File: modules/acquia_cms_common/tests/src/ExistingSite/ContentTypeListTestBase.php [hunk 4]
```diff
@@ -382,7 +382,7 @@ protected function getQuery() : QueryInterface {
   /**
    * {@inheritdoc}
    */
-  public function tearDown() {
+  public function tearDown() : void {
```
Comment (apathak18) [inline modules/acquia_cms_common/tests/src/ExistingSite/ContentTypeListTestBase.php:385]: @deepakmishra2 Covered in this please remove from this change ~~ https://github.com/acquia/acquia_cms/pull/1356/files#diff-1b1f03796fd04b1a3ea69c3a7b0054be5134cda91f94bdb4d7661ffa7408a4dbR385

cc: @rajeshreeputra

**Review by rajeshreeputra — APPROVED** [review]
LGTM

**Review by apathak18 — APPROVED** [review]
LGTM

**Pull request merged**